### DISCUSSION TITLE
Compile in Windows without unix-like command-line

### DISCUSSION THREAD

**Discussion by superpi15** [YosysHQ/yosys#3095] [Q&A] [answered]
Hi friends, 
First of all, I'm a unix guy. Yosys works perfectly well in unix-like system.
But I'm wondering, how to compile Yosys in Windows? 
My guess is, set the required packages with Cygwin then things become easier. 
Would it be possible to compile Yosys with Visual studio? 
Any hints or comments are welcome!

**Comment by nakengelhardt** [comment, marked as answer]
Yes, it is possible to compile with Visual studio. I'm not sure how up to date the [guideline](https://github.com/YosysHQ/yosys/blob/master/guidelines/Windows) is, but we have it running in CI so you can compare with the [github action](https://github.com/YosysHQ/yosys/blob/master/.github/workflows/vs.yml).

**Reply by superpi15** [reply]
Thank you very much for the response. 
I will spread your words.
